test/common/upstream/subset_lb_test.cc

```diff
@@ -161,6 +161,38 @@ class TestMetadataMatchCriteria : public Router::MetadataMatchCriteria {
   std::vector<Router::MetadataMatchCriterionConstSharedPtr> matches_;
 };
 
+void SubsetLoadBalancerInternalStateTester::testWrapper() {
+  MockLoadBalancerContext inner;
+  TestMetadataMatchCriteria test_criteria(std::map<std::string, std::string>{});
+
```

Comment:
I think I had added this test in the #27593.

See the file test/extensions/load_balancing_policies/subset/subset_test.cc